src/cli.js

```diff
@@ -8,6 +8,18 @@ import { version } from '../package.json';
 import apiUrls from '../config/services';
 import autoUpdate from './commands/update-cli';
 import * as analytics from './extension/analytics';
+import { isAscii, containsSpace } from './extension/validation';
+import getHomeDir from './home-dir';
+
+const cliReferenceUrl = 'https://shoutem.github.io/docs/extensions/reference/cli';
+
+const homeDir = getHomeDir();
+if (!isAscii(homeDir) || containsSpace(homeDir)) {
```

Comment:
Because there's nothing async in it.
I don't like indenting code if I don't have to :)